install.sh

```diff
@@ -106,7 +106,7 @@ appendShellPath() {
   fi
 
   local zshrc_file="${HOME}/.zshrc"
-  if [ -f "${zshrc_file}" ]; then
+  if [ -f "${zshrc_file}" ] || [ "${OS}" = "darwin" ]; then
```

Comment:
Should we move this block after fish check?
Can someone have both MacOS + fish?

Reply:
yes someone can have both macos and fish, in this case we will append paths on both configs